go/protocols/flow/flow.proto

```diff
@@ -461,11 +461,14 @@ message OAuth2Spec {
   // where we request an access token from the provider
   string access_token_url_template = 3;
 
+  // The method used to send access_token request. POST by default.
+  string access_token_method = 4;
+
   // The POST body of the access_token request
-  string access_token_body = 4;
+  string access_token_body = 5;
```

Comment:
Renumbering proto fields always makes me nervous. Are we confident there's no way the sender and recipient of this proto could be out of sync?

Reply:
My understanding is that this OAuth2 flow is not used outside of this repository at the moment, the agent in animated-carnival considers it a Raw JSON value so we should be good